java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -199,4 +199,26 @@ public interface StringBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> strlen(String key);
+
+    /**
+     * Overwrites part of the string stored at <code>key</code>, starting at the specified <code>
+     * offset</code>, for the entire length of <code>value</code>.<br>
+     * If the <code>offset</code> is larger than the current length of the string at <code>key</code>,
+     * the string is padded with zero bytes to make <code>offset</code> fit. Creates the <code>key
+     * </code> if it doesn't exist.
+     *
+     * @see <a href="https://redis.io/commands/setrange/">redis.io</a> for details.
+     * @param key The key of the string to update.
+     * @param offset The position in the string where <code>value</code> should be written.
+     * @param value The string written with <code>offset</code>.
+     * @return The length of the string stored at <code>key</code> after it was modified.
+     * @example
+     *     <pre>{@code
+     * long len = client.setrange("key", 6, "GLIDE").get();
```

Comment:
Shouldn't it be Long?

Reply:
Please make sure that the examples compile

Reply:
Updated. Both are ok and compilable